business_logic/participant_auth_utils/participantAuthUtils.js

```diff
@@ -0,0 +1,74 @@
+const twitterComm = require("../twitter_communicator/twitterCommunicator")
+const manipulator = require("../manipulator/manipulator.js")
+const database = require("../db/DBCommunicator.js");
+const { data } = require("../twitter_communicator/static_twitter_data/FeedJSON");
```

Comment:
לא צריך כאן את מניפולטור וה feed json

Reply:
סעמק אני עורך מימין לשמאל וזה הופך את הכיוון אחרי פרסום התגובה.
No need for manipulator and feedJson here